dataGeneration/requirements.txt

```diff
@@ -0,0 +1,5 @@
+numpy==1.21.2
```

Comment:
how do we use this file? Or is it just documentation?

Reply:
we can use `pip install -r requirements.txt` will add a readme with that info

Reply:
Any requirement for python version?

Reply:
I am using 3.7 only so I stated that as required, not sure how to know what is minimum without manual checking